### PR TITLE
basecss linked and order css linked, needs additional tweaking 

### PR DESCRIPTION
**Pull request by bizzybits**
and additional clarification on formatting for forms

### CODE REVIEW

File: orderpage.html [hunk 1]
```diff
@@ -42,10 +38,7 @@ <h3>Select up to two jams:</h3>
                 <input type="checkbox" name="jams" value="Marionberry"/> Marionberry
                 <input type="checkbox" name="jams" value="Orange Marmalade"/> Orange Marmalade
                 <input type="checkbox" name="jams" value="Mixed Berry"/> Mixed Berry
-          </legend>
-        </fieldset>
-        <fieldset>
-          <legend>
+
             <h3>Want Extras?</h3>
               <br/>
```
Comment (Katedam) [inline orderpage.html:56]: Just a reminder, you don't need break tags unless you're in the middle of a paragraph or just after an inline tag. All h1-6 tags, inputs, p, etc. are block tags so they start a new a new line anyways :-)

File: orderpage.html [hunk 2]
```diff
@@ -17,10 +19,7 @@ <h3>Choose your bread:</h3>
             <input type="radio" name="breads" value="Bagel"/> Bagel
             <input type="radio" name="breads" value="Teff Wrap"/> Teff Wrap without Honey
             <input type="radio" name="breads" value="English Muffin"/> English Muffin
-          </legend>
-        </fieldset>
-        <fieldset>
-          <legend>
+
             <h3>Select up to two spreads:</h3>
               <p>
```
Comment (Katedam) [inline orderpage.html:37]: FYI this p tag isn't needed, the inputs can hang on their own, thanks boo

File: orderpage.html [hunk 3]
```diff
@@ -2,12 +2,14 @@
...
+    <link rel="stylesheet" type="text/css" href="css/base.css">
+    <link rel="stylesheet" type="text/css" href="css/order.css">
   </head>
     <body>
       <h1>Make Sandwich!</h1>
         <h3 id="userName"></h3>
         <form action="https://www.i.think.this.is.where.we.fudge.com">
-        <fieldset>
+
           <legend>
```
Comment (Katedam) [inline orderpage.html]: quick review on the legend tag! It's kind of an odd thing, it goes in the border that the fieldset tag throws around the inputs but you don't actually have to wrap the inputs with it. So keep the opening and closing tags together on one line and put whatever title for that form you want inside, like so: <legend>Jams!</legend>
Then go to the next line and add your inputs. They all indent the same amount too 👍

**Review by Katedam — CHANGES_REQUESTED** [review]
Hey, once you make these updates, you can just go back to the pull request and do the merge yourself - no need to wait again! Thanks!!

File: orderpage.html (hunk 2 above)
Comment (bizzybits) [reply orderpage.html:37]: I can't figure out how to get the css to work if i don't leave the p tag? I tried Input with the same properties and it didn't work?

**Pull request merged**